src/app/test/dot-themes.mock.ts

```diff
@@ -0,0 +1,36 @@
+import { DotTheme } from '../portlets/dot-edit-page/shared/models/dot-theme.model';
+
+export const mockDotThemes: DotTheme[] = [
```

Comment:
I think is better idea define this object into DotThemesServiceMock files and not export it, in this way we can be sure that the only way to use this object is through the service so we always would use a copy of this object into each test